core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityManager.scala

```diff
@@ -111,12 +112,16 @@ object EntityManager {
                            cacheEnabled: Boolean,
                            queryTimeout: Duration,
                            metricsPrefix: String
-  )(implicit ec: ExecutionContext): EntityManager = {
+  )(implicit ec: ExecutionContext, system: ActorSystem): EntityManager = {
     // create the EntityManager along with its associated provider-builders. Since entities are only accessed
     // in the context of a workspace, this is safe/correct to do here. We also want to use the same dataSource
     // and execution context for the rawls entity provider that the entity service uses.
     val defaultEntityProviderBuilder =
-      new LocalEntityProviderBuilder(dataSource, cacheEnabled, queryTimeout, metricsPrefix) // implicit executionContext
+      new LocalEntityProviderBuilder(dataSource,
+                                     cacheEnabled,
+                                     queryTimeout,
+                                     metricsPrefix
+      ) // implicit executionContext, system
```

Comment:
`LocalEntityProvider` now needs the `ActorSystem` which is created in `Boot` - this cascades changes through multiple files in this PR.